bodhi/client/bindings.py

```diff
@@ -103,10 +118,10 @@ def __unicode__(self):
     __str__ = __unicode__
 
 
-def errorhandled(method):
+def errorhandled(method: typing.Callable) -> typing.Callable:
     """Raise exceptions on failure. Used as a decorator for BodhiClient methods."""
     @functools.wraps(method)
-    def wrapper(*args, **kwargs):
+    def wrapper(*args, **kwargs) -> typing.Any:
         try:
             result = method(*args, **kwargs)
```

Comment:
Since Bodhi is using Pyhon3.6, and [annotating variables](https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html#variables) was introduced in Python 3.6, we can use it for variables.
What do you think?

Reply:
@sebwoj oh nice, I didn't know about that. I don't think we need to annotate all variables since mypy can infer types in most cases, but I'll adjust the places I used the comment to do it this way.

Reply:
I'll apply this patch:

```
diff --git a/bodhi/client/bindings.py b/bodhi/client/bindings.py
index f77aeeaa..11f85ab5 100644
--- a/bodhi/client/bindings.py
+++ b/bodhi/client/bindings.py
@@ -530,7 +530,7 @@ class BodhiClient(OpenIdBaseClient):
             page (int): Return a specific page of results.
                 (default:None)
         """
-        params = {}  # type: typing.MutableMapping[str, typing.Union[int, str, None]]
+        params: typing.MutableMapping[str, typing.Union[int, str, None]] = {}
         if user:
             params['user'] = user
         if packages:
```